crrExistingObjects.js

```diff
@@ -80,7 +84,23 @@ const options = {
         agent: new http.Agent({ keepAlive: true }),
     },
 };
-const s3 = new AWS.S3(options);
+/**
+ *  Options specific to s3 requests
+ *  `maxRetries` & `customBackoff` are set only to s3 requests
+ *  default aws sdk retry count is 3 with an exponential delay of 2^n * 30 ms
+ *  aws sdk retries are used first, before trying the requests recursively
+ *  on errors!
+ */
+const s3Options = {
+    maxRetries: AWS_SDK_REQUEST_RETRIES,
+    customBackoff: (retryCount, error) => {
+        log.debug('AWS sdk request error', { error, retryCount });
+        // computed delay is not truly exponential, it is reset to minimum after
```

Comment:
What is the rationale to reset to the minimum after 10 calls? Usually when we consider the retry delay to be too large, we keep it to a maximum value.

Reply:
The delay is too long if we don't repeat in cycles, with the current setup every tenth call takes 15 seconds to retry, this could be a tunable as well. I thought it would be better if we decide what the maximum delay could be.